src/ore/src/operators_async_ext.rs

```diff
@@ -102,12 +106,16 @@ pub trait OperatorBuilderExt<G: Scope> {
     /// ecosystem the only way to yield control back to timely is by awaiting on
     /// `scheduler.yield_now()`. The operator will ensure that this call resolves when there is
     /// more work to do.
+    ///
+    /// The `reschedule_flag` argument to the `constructor` is used to communicate whether the
+    /// operator is complete.  It corresponds to the `bool` return value of `build_reschedule`.
     fn build_async<B, Fut>(self, scope: G, constructor: B)
     where
         B: FnOnce(
             Vec<Capability<G::Timestamp>>,
             Rc<RefCell<Vec<Antichain<G::Timestamp>>>>,
             Scheduler,
+            Rc<RefCell<bool>>,
```

Comment:
General rust tip: for simple copy types like `bool` an `Rc<Cell<bool>>` is usually enough if you just want to treat it as a shared mutable memory location